.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -41,9 +41,14 @@ jobs:
         instance-type:
           - small
           - normal
+        kbs:
+          - false
         include:
           - host_os: cbl-mariner
             vmm: clh
+          - kbs: true
+            vmm: qemu
+            instance-type: small
```

Comment:
Now that I'm thinking about it, it'll change the name of the test, which will require the dance of making tests non-required, and asking folks to rebase.

I'm okay with it, but I'd like to hear from @stevenhorsman on this.

Reply:
I think it'd be clearer to have the logic inside the shell scripts (with a possibility to override things) as the need for kbs comes from the test suite rather than from the workflow (although the line is pretty thin as the workflow defines the suite). So +1 for having it as part of the shell scripts from me.

Reply:
I think that having that as part of the shell scripts simply goes against what we've been doing as part of the Ci.
We've defined whether or not we install devmapper or nydus-snapshotter as part of the action, this one would be inconsistent.

Reply:
I see, TBH I'm not that familiar with the kata-containers style and I reacted based on my expectations (rather than magic variable I tend to favour logic to enable dependencies and kbs seems like a dependency, rather than module, to me)

Reply:
@fidencio one option to avoid the jobs renaming is to remove the new matrix entry; we filter by `vmm` on the `if` conditional of the step that install the KBS. All in all, we want to install kbs to run additional tests, if we create a new jobs we will be re-running tests already covered (increasing the odds of intermittent failures).